package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript-ocelot",
-  "version": "0.14.5",
+  "version": "0.14.6",
```

Comment:
Can we publish this to bower once it's approved? Will solve #77

Reply:
Yep.

Reply:
Oh yeah, someone in Slack mentioned Ocelot doesn't have a license, let's add that as well before tagging and publishing. I can do it when I get to the office later.